### ISSUE TITLE
Generalize `SPIKE_GENE_COVERAGE` to cover all genes

### ISSUE THREAD

**Issue by fanninpm** [usafsam/mad_river_wf#10]
Currently, `nextalign_gene_coverage.R` only focuses on one gene region at a time. I'd like to rework that script so it can take a GFF file as the source for all of its genes.

**Comment by friesac** [comment]
Good idea, along with tabular output of the performance it would be beneficial to have a visual quick check of any single segment and the entire genome. I like the segment coverage figures like IRMA gives at [coverageDiagram.pdf](https://wonder.cdc.gov/amd/flu/irma/output.html). It would be tough.

Probably not in the scope of this issue but would also be most beneficial to have a number of reads per amplicon as defined by the .bed file. i.e. we have been seeing massive failure of the 28th amplicon fragment of the midnight primers with omicron but we only truly see that when we look at the full consensus and see "NNNNN" in that region.

**Comment by fanninpm** [comment]
The formula in https://github.com/nextstrain/nextclade/pull/956 might be interesting to look at.
